### PR TITLE
makes paper bins use incapacitated() instead of a massive if()

### PR DESCRIPTION
**Pull request by PKPenguin321**
i can into cleaning code
hopefully this works

### CODE REVIEW

**Comment by PKPenguin321** [comment]
i promise the title is a typo the code should be fine (I HOPE :cry: )

File: code/modules/paperwork/paperbin.dm [hunk 1]
```diff
@@ -25,7 +25,7 @@
 
 /obj/item/weapon/paper_bin/MouseDrop(atom/over_object)
 	var/mob/M = usr
-	if(M.restrained() || M.stat || !Adjacent(M))
+	if(M.incapacitated())
```
Comment (KorPhaeron) [inline code/modules/paperwork/paperbin.dm]: This lacks an adjacency check

Comment (PKPenguin321) [inline code/modules/paperwork/paperbin.dm]: shoot

**Comment by neersighted** [comment]
:+1:

**Pull request merged**